Studies/notebooks/SignalSelectionNNTraining.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re \n",
+    "import os\n",
+    "import tensorflow as tf\n",
+    "import ROOT\n",
+    "\n",
+    "from tensorflow.keras.callbacks import CSVLogger\n",
+    "\n",
+    "import sys\n",
+    "sys.path.insert(0, \"../python\")\n",
+    "import InputsProducer\n",
+    "from SignalSelectionModel import HHModel"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ListToVector(l, elem_type):\n",
```

Comment:
Define function `FindFiles(path, pattern)` in InputsProducer and move code from this cell and the cell below there.